### PR TITLE
Adding proper color variables and fixing the color backgrounds

### PR DESCRIPTION
**Pull request by znegrin**
As requested on the issue #73 I included here the proper color variables and fixed the colors for the pipeline elements removing the previous active behavior and replacing it with css styles.

image

image

image

image

Signed-off-by: Zuri Negrín <zurinegrin@gmail.com>

### CODE REVIEW

File: src/sass/_overrides.scss [hunk 1]
```diff
@@ -44,3 +44,29 @@ $font-family-sans-serif: 'Cerebri', -apple-system, BlinkMacSystemFont, "Segoe UI
...
+
+$colors: (
+        "blue": #147EEC,
+        "purple": #9260E2,
+        "pink": #FC1763,
+        "red": #EF3837,
+        "orange": #FF7427,
+        "yellow": #FFC508,
+        "green": #2FCC71
+) !default;
```
[the remark this answers is not in the document]
Comment (znegrin) [reply src/sass/_overrides.scss]: I actually followed the Bootstrap documentation, but if you think it is not necessary I can get rid of it.

**Comment by znegrin** [comment]
> Could you please also update [`src/js/res/palette.js`](https://github.com/athenianco/athenian-webapp/blob/master/src/js/res/palette.js) according to the new color spec?
> Also, delete `*Alt` versions (from this `palette.js` same file, which are not needed, thanks.

I fixed this one.

File: src/sass/_overrides.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (dpordomingo) [reply src/sass/_overrides.scss]: From my understanding it makes sense for bootstrap, but not for our app (unless we're planning to white brand the webapp to be used and styled by others).
But I might be missing something. Could you please share a link to that docs so I could learn more?

[the remark this answers is not in the document]
Comment (znegrin) [reply src/sass/_overrides.scss]: I was checking this page, but I think I was wrong as I did it before digging better into the structure of the project:

https://getbootstrap.com/docs/4.1/getting-started/theming/#theme-colors

[the remark this answers is not in the document]
Comment (dpordomingo) [reply src/sass/_overrides.scss]: yes, so `!default` [is only recommended for them](https://getbootstrap.com/docs/4.1/getting-started/theming/#variable-defaults) in this case.
So let's remove it, and done :D

**Comment by dpordomingo** [comment]
ping me when you remove `!default` and fix conflicts and I'll merge.

**Comment by znegrin** [comment]
@dpordomingo I fixed the merge conflicts and removed the `!default`.

**Pull request merged**